src/main/java/uk/gov/hmcts/dts/fact/entity/CourtAreaOfLawSpoe.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.dts.fact.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "search_courtareaoflawspoe")
+@Getter
+@Setter
+@NoArgsConstructor
+public class CourtAreaOfLawSpoe {
+    @Id
+    @SequenceGenerator(name = "seq-gen", sequenceName = "search_courtareaoflaw_id_seq", allocationSize = 1)
```

Comment:
Might be worth renaming the sequenceName to search_courtareaoflawspoe_id_seq reflecting the table name